wiki/People/Wiki_Maintainers/en.md

```diff
@@ -0,0 +1,16 @@
+## Wiki Maintainers
+Wiki Maintainers are a group of players who are in charge of maintaing the wiki and merging changes into the offical Titanic! wiki.
+
+## Roles and Responsibilities
+Wiki Maintainers are responsible for merging new pull requests and making sure there are no errors.
+
+## Current Members
+
+User | Added Date 
+---|---
+[BlueChinchompa](https://osu.titanic.sh/u/40)   | Since The Beginning 
```

Comment:
Levi should know about this info, it would be best to wait for review

Reply:
I agree I tried searching around in the `#Wiki` channel in discord and any mention of maintainer/invite but I didn't find anything.

Reply:
oh wait, you may find it on your email address which is connected to your github, perhaps try to find it on gmail or something

Reply:
It would be on your gmail, yeah